integration-testing/build.gradle

```diff
@@ -93,6 +105,14 @@ task debugAgentTest(type: Test) {
     systemProperties project.properties.subMap(["overwrite.probes"])
 }
 
+task debugDynamicAgentTest(type: Test) {
+    environment "version", coroutines_version
```

Comment:
Could you drop a comment on why this is needed?

Reply:
oh, this one is indeed not needed, I'll remove it